frontend/requests/requestDialogService.js

```diff
@@ -77,6 +77,8 @@
                 case REQUEST_PARENT:
                 case REQUEST_CHILDREN:
                     service.showHierarchyDialog(request, event); break;
+                case REQUEST_INSTITUTION:
```

Comment:
I do not understand why this is case necessary, once the default case has the same action and the REQUEST_INSTITUTION type should match with it.

Reply:
Because this approach is more readable and let it clearer that this case is treated in there.